check_tv.py

```
@@ -75,17 +75,15 @@ def offline_check(driver, live_url, spare_link, important):
       while True:
          try:
              current_url = driver.current_url
-             if config.Twitch == "True":
-               if current_url == desired_url:
+             if config.Twitch == "True" and current_url == desired_url:
                  ok = "ok"
-             if config.BiliBili == "True":
-                 if current_url == bilibili_desired_url:
+
+             if config.BiliBili == "True" and current_url == bilibili_desired_url:
                        ok = "ok"
```

Ruff (0.8.2)

Local variable `ok` is assigned to but never used

Remove assignment to unused variable `ok`

(F841)